db/controller.js

```diff
@@ -16,3 +20,12 @@ exports.getArticles = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.sendArticlesById = (req, res, next) => {
```

Comment:
I would maybe change this function name, as `send` suggests that we're sending something to the database instead of getting something from it